vagrant_devstack.md

```diff
@@ -0,0 +1,61 @@
+# Installing Romana on Vagrant with Devstack
+
+The setup described below has been tested on Ubuntu 14.04 LTS, but should work similarly on other Linux or Mac OS X environments.
+You may need to install additional development tools.
+
+This should be done on your host. A 'run from a VM' option is being developed, for users that do not wish to install additional tools on the host.
+
+# Prepare
+
+To run this installation, you will need
+* [Vagrant](https://www.vagrantup.com/downloads.html) installed (and [tested](https://www.vagrantup.com/docs/getting-started/) to be sure it works)
+* [ansible](https://www.ansible.com) and supporting python tools / libraries
+
+## Set up Ansible
+
+```bash
+# Ubuntu
+sudo apt-get install git python-pip python-dev
+sudo pip install ansible==1.9.4 netaddr
+
+# OS X
+sudo easy_install pip
+sudo pip install ansible==1.9.4 netaddr
+```
+
+# Install
+
+Check out the Romana repository and run the installer
+```bash
+git clone https://github.com/romana/romana
+cd romana/romana-install
+./romana-setup -p vagrant -s devstack install
+```
+
+The Vagrant installation can take a long time to complete, because of some large downloads that are performed. Please be patient. When installation is complete, information about the cluster should be provided.
```

Comment:
vagrant image, then apt-get update, apt-get upgrade, apt-get install a bunch of things, kubernetes download, etcd download, romana core binaries, romana third-party binaries..on each VM. and finally the container images/layers.
It's a bit weighty, so if your bandwidth is a bit small, the install is going to be slow.

A cache would be handy on the N'th install, not so much for the first.